README.rst

```diff
@@ -125,7 +134,15 @@ To enable themes in your Tutor environment, follow these steps:
     # Enable themes
     tutor picasso enable-themes
 
-This command clones your theme repository into the folder that Tutor uses for themes. Documentation available at `Installing custom theme`_ tutorial.
+This command will clone your theme repository into the folder that Tutor uses for themes. You can find the documentation in the `Installing custom theme`_ tutorial.
+
+If ``PICASSO_THEMES`` is defined, the plugin will set ``ENABLE_COMPREHENSIVE_THEMING = True``.
+
+If ``PICASSO_THEME_DIRS`` is defined, the plugin will extend the ``COMPREHENSIVE_THEME_DIRS`` by patches.
+
+The ``PICASSO_TUTOR_VERSION``, ``PICASSO_THEME_DIRS`` and ``PICASSO_THEMES_NAME`` variables are used to compile the themes. For detailed information, see the patch `openedx-dockerfile-pre-assets <tutorpicasso/patches/openedx-dockerfile-pre-assets>`_.
```

Comment:
I am trying to understand why we need another tutor version variable. Could you explain more about this? Thanks.

Reply:
We need the tutor version defined to choose the patch to use https://github.com/eduNEXT/tutor-contrib-picasso/blob/mfmz/fix-themes/tutorpicasso/patches/openedx-dockerfile-pre-assets#L8

I proposed changing the name of the `TUTOR_VERSION` variable we use in Picasso (job/action) to convert it into a variable for this plugin with the prefix. But it is the same. If we don't want to make more changes, we can use the same `TUTOR_VERSION`.
I proposed the `PICASSO_TUTOR_VERSION` because this plugin will need that variable and follow the good practice of variables with the prefix for a variable to be unique.

Reply:
Oh, now I understand. My concern is that we already have a variable referring to the tutor version, so I wonder why we'd need a new one. What would happen if `TUTOR_VERSION`, for some reason, is different than `PICASSO_TUTOR_VERSION`? Is `PICASSO_TUTOR_VERSION` defaulting to `TUTOR_VERSION`? Let me know. Thanks!

Reply:
Also, TUTOR_VERSION follows this format: [("TUTOR_VERSION", __version__),](https://github.com/overhangio/tutor/blob/46b401690c812448c17b2cd91f90cf765a7a9678/tutor/env.py#L66-L67) which corresponds to X.Y.Z instead of vX.Y.Z https://github.com/overhangio/tutor/blob/46b401690c812448c17b2cd91f90cf765a7a9678/tutor/__about__.py#L3-L5

Reply:
Thanks @MaferMazu, the code looks good to me. However, I agree on using `TUTOR_VERSION`, I tested it and works

Reply:
I applied the changes to use `TUTOR_VERSION`. Thanks for the feedback.

About why we have our TUTOR_VERSION starting with v is for TVM. When we create the projects, we use something like: `tvm project init my_project v18.1.3`. I heard something about not using TVM in our build process, so I am unsure if we still need that v, but we can start that conversation later to implement that change and ensure everything works fine.

Reply:
Sounds good to me. Thank you!